packages/test/test-pairwise-generator/README.md

```diff
@@ -8,7 +8,7 @@ See [examples.spec.ts](./src/test/examples.spec.ts) for usage examples.
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
 **NOTE: This package is private to the `@microsoft/fluid-framework` repository.**
-**It is not published, and therefore may only be used as a dev dependency.**
+**It is not published, and therefore may only be used as a dev dependency in packages within the same pnpm workspace in this repo using the `workspace:*` schema.**
```

Comment:
I'm actually now questioning if devDependency only is correct. test-drivers uses it as a prod dep. Maybe say it must be a dev-dep for public packages, and for others (like e2e tests and stress tests) it can be a prod dep?

Reply:
Ah, yes good point. I'll expand these.

Reply:
Updated. Let me know what you think.